babel.config.js

```diff
@@ -1,5 +1,3 @@
 module.exports = {
-  presets: [
-    '@vue/app',
-  ],
+  presets: [['@vue/app', { useBuiltIns: 'entry' }]],
```

Comment:
Why do you need `{ useBuiltIns: 'entry' }`?

Reply:
I just copied this from [here](https://github.com/vuejs/vue-cli/issues/3678) as a solution of my build problem and it`s works.

Reply:
Ok but I think this fix should be in separate branch ;)

Reply:
I removed this change